content/posts/weekly/February-2023.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "February 2023"
+title: "Packit February 2023"
```

Comment:
:+1: can someone actually verify this properly displays in RSS readers?

Reply:
Yes, it does directly take the title. Check this out: https://packit.dev/posts/weekly/index.xml :)

Reply:
But sadly, the content still does not contain all the weeks... but that's a different issue.